composer.json

```diff
@@ -25,17 +25,25 @@
         }
     ],
     "require": {
-        "php": "^5.4|^7.0",
-        "tedivm/stash": "0.14.*",
-        "symfony/config": "~2.1|~3.0",
-        "symfony/http-kernel": "~2.1|~3.0",
-        "symfony/dependency-injection": "~2.1|~3.0"
+        "php": "^7.0",
+        "tedivm/stash": "^0.15",
+        "symfony/config": "^3.4",
+        "symfony/http-kernel": "^3.4",
+        "symfony/dependency-injection": "^3.4"
```

Comment:
on the 3 packages above, did you intend to say  `"^3.4 | ^4.0"` ? _(given your PR description)_

Reply:
My intention was to make the code ready for 4.0, as soon as we upgrade our plaform to this SF version I intended to upgrade the packages to 4.0. This because I can guarantee the working then (if will have seen it working on our platform with millions of requests per day). But I guess we can do it this way also! Given the fact that the code will be 4.0 deprecation free this should word (famous last words) :stuck_out_tongue: :+1:

Reply:
Exactly :)